docs/Contributors_Guide/continuous_integration.rst

```diff
@@ -1,115 +1,376 @@
+**********************
 Continuous Integration
-======================
-
-More information on Continuous Integration (CI) coming soon!
+**********************
 
 METplus utilizes GitHub Actions to run processes automatically when changes
 are pushed to GitHub. These tasks include:
 
-* Building documentation
+* Building documentation to catch warnings/errors
 * Building a Docker image to run tests
 * Creating/Updating Docker data volumes with new input data used for tests
 * Running unit tests
 * Running use cases
 * Comparing use case output to truth data
 * Creating/Updating Docker data volumes with truth data to use in comparisons
 
-Workflow Control
-----------------
+GitHub Actions Workflows
+========================
 
-GitHub Actions is controlled by a file in the .github/workflow directory called
-testing.yml. If this file exists and is valid (no errors), GitHub Actions will
-read this file and trigger a workflow run if the triggering criteria is met.
+GitHub Actions runs workflows defined by files in the **.github/workflows**
+directory of a GitHub repository.
+Files with the .yml suffix are parsed and GitHub Actions will
+trigger a workflow run if the triggering criteria is met.
 It can run multiple jobs in parallel or serially depending on dependency rules
 that can be set. Each job can run a series of commands or scripts called steps.
 Job steps can include "actions" with can be used to perform tasks. Many useful
 actions are provided by GitHub and external collaborators. Developers can also
 write their own custom actions to perform complex tasks to simplify a workflow.
 
```

Comment:
Consider noting here that a single push can result in multiple workflows being triggered.

All workflow runs can be seen on the Actions tab of the repository. Each workflow run is identified by the branch for which it was invoked as well as the corresponding commit message on that branch. In general, a green check mark indicates that all checks for that workflow run passed. A red X indicates that at least one of the workflow checks failed.

Any workflow runs associated with a PR can also be seen on the "Checks" tab of that PR.

Consider adding 1 or 2 images to illustrate.